websites/A/Airbnb/presence.ts

```diff
@@ -0,0 +1,49 @@
+const presence = new Presence({
+		clientId: "993151156652093480",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "logo",
+		startTimestamp: browsingTimestamp,
+	};
+
+	if (document.location.pathname.includes("/rooms")) {
+		presenceData.details = "Viewing rooms";
```

Comment:
This suggests that the user is viewing multiple rooms, whereas the button contradicts this and implies that the user is viewing a singular room.